staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -5276,7 +5276,8 @@ type DownwardAPIVolumeSource struct {
 	// +optional
 	Items []DownwardAPIVolumeFile `json:"items,omitempty" protobuf:"bytes,1,rep,name=items"`
 	// Optional: mode bits to use on created files by default. Must be a
-	// value between 0 and 0777. Defaults to 0644.
+        // Optional: mode bits used to set permissions on created files by default. Must be an
```

Comment:
```suggestion
      // Optional: mode bits used to set permissions on created files by default. Must be an
```
Small nit pick, remove two leading spaces at start of this comment.